exec/plugin-commands-script-runners/test/runRecursive.ts

```diff
@@ -1059,41 +1059,3 @@ test('pnpm recursive run report summary with --bail', async () => {
   expect(executionStatus[path.resolve('project-4')].status).toBe('queued')
   expect(executionStatus[path.resolve('project-5')].status).toBe('skipped')
 })
-
-// TODO: fix the related implementation then remove `.skip`
-test.skip('recursive run when some packages define node version, config does not define use-node-version', async () => {
-  preparePackages([
-    {
-      name: 'node-version-undefined',
-      scripts: {
-        'assert-node-version': `node -e "assert.equal(process.version, '${process.version}')"`,
-      },
-    },
-    {
-      name: 'node-version-18',
-      scripts: {
-        'assert-node-version': 'node -e "assert.equal(process.version, \'v18.0.0\')"',
-      },
-      pnpm: {
-        useNodeVersion: '18.0.0',
-      },
-    },
-    {
-      name: 'node-version-20',
-      scripts: {
-        'assert-node-version': 'node -e "assert.equal(process.version, \'v20.0.0\')"',
-      },
-      pnpm: {
-        useNodeVersion: '20.0.0',
-      },
-    },
-  ])
-  await run.handler({
-    ...DEFAULT_OPTS,
-    ...await filterPackagesFromDir(process.cwd(), [{ namePattern: '*' }]),
-    dir: process.cwd(),
-    recursive: true,
-    workspaceDir: process.cwd(),
-    workspaceConcurrency: 3, // if this step fails, change this value to 1 to see error message.
-  }, ['assert-node-version'])
-})
```

Comment:
This test doesn't work when placed here but it work when moved to `/pnpm/test`. Not sure why.